code/modules/vehicles/wheelchair.dm

```diff
@@ -14,6 +14,14 @@
 	var/image/wheels_overlay
 	///Determines the typepath of what the object folds into
 	var/foldabletype = /obj/item/wheelchair
+	///Determines whether the wheelchair has a bell on it or not
+	var/bell_attached
```

Comment:
```suggestion
	///Bell attached to the wheelchair, if we have one.
	var/obj/structure/desk_bell/bell_attached
```
This will let you do more stuff with it as well, like making the drop not just new another one.